community/modules/scripts/wait-for-startup/main.tf

```diff
@@ -13,22 +13,22 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-data "google_compute_instance" "vm_instance" {
```

Comment:
It would still be useful to have this data source to keep `triggers` around.

Reply:
I understand why it was useful when we wanted to trigger 1 script per machine, but now 1 instance of the script handles all machines.  How would we use triggers in this case?